2021/burger2022reproducibility/openreview/packages.tex

```diff
@@ -0,0 +1,25 @@
+% Import packages here
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[]{neurips_2019}
+
+% to avoid loading the natbib package, add option nonatbib:
+%     \usepackage[nonatbib]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+%\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+
+\usepackage{subcaption}
```

Comment:
From the build logs, the first error is:

> ! Package subcaption Error: This package can't be used in cooperation
> (subcaption)                with the subfig package.

I would recommend removing the subcaption package.

Reply:
Hi, thanks for the pointer! After removing the package, the figures and tables are now displayed correctly. Unfortunately, we still face the issue of undefined citations and references and the header overlapping. Do you have any clue what could cause this?